catalog/Podfile

```diff
@@ -53,11 +89,10 @@ post_install do |installer|
   # Inject our specific warning flags into the .xcconfig files.
   mdc_xcconfigs = [
     "#{pod_dir}/Target Support Files/MaterialComponents/MaterialComponents.xcconfig",
-    "#{pod_dir}/Target Support Files//MaterialComponentsExamples/MaterialComponentsExamples.xcconfig",
+    "#{pod_dir}/Target Support Files/MaterialComponents/MaterialComponents.unit.xcconfig",
```

Comment:
Is this going to need MaterialComponentsAlpha.unit as well?

Reply:
Good catch!